discovery/kubernetes/kubernetes.go

```diff
@@ -167,7 +170,11 @@ func (c *SDConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
 	if err != nil {
 		return err
 	}
-	if c.APIServer.URL == nil && !reflect.DeepEqual(c.HTTPClientConfig, config.DefaultHTTPClientConfig) {
+	if c.APIServer.URL != nil && c.KubeConfig != "" {
+		// for api-server and kubeconfig_path are mutually exclusive
```

Comment:
```suggestion
		// Api_server and kubeconfig_file are mutually exclusive.
```

We use full sentences in comments.

Reply:
ok, I fix it